utils/convert-time.js

```diff
@@ -0,0 +1,18 @@
+function convertTime(num) {
+    const number = num;
+    const hours = num / 60;
+    const roundHours = Math.floor(hours);
+    const minutes = (hours - roundHours) * 60;
+    const roundMinutes = Math.round(minutes);
+
+    if (number < 60) {
+        return `${roundMinutes} minutes`
+    } else if (number === 60) {
+        return `${hours} hour`;
+    } else if (number > 60) {
+        return `${roundHours} hours and ${roundMinutes} minutes`
+    }
+
+  }
+
```

Comment:
I like how this work. The only part that I found a bit confusing and potentially repetitive was line 2:

const number = num;

Could num be used everywhere that number is used?

Reply:
Updated.